lib/db/mysql/index.js

```diff
@@ -177,7 +195,56 @@ MysqlStore.prototype = {
       return client;
     });
   },
+  registerClientDeveloper: function regClientDeveloper(developerId, clientId) {
+    if (!developerId || !clientId) {
+      var err = new Error('Owner registration requires user and developer id');
+      return P.reject(err);
+    }
+
+    logger.debug('registerClientOwner: ', developerId, clientId);
```

Comment:
This format is illegal for heka. The first argument must be a string of no spaces (probably no colons either). The second argument should be either 1 value, or an object mapping names and values.